recipe/meta.yaml

```diff
@@ -18,8 +18,12 @@ source:
   fn: cmake-{{ version }}-win64-x64.zip                                                   # [win64]
   sha256: 659ecb8207e1266786188c7eaf45308458ba5f719c985970f6f55ec0b5a96746                # [win64]
 
+  patches:
+    - FindBoost-Add-support-for-1.63.patch  # [not win]
```

Comment:
Ok, I've added the link inside the patch header, but as a comment in the `yaml` seems more visible!